src/CMake/FindADIOS2.cmake

```diff
@@ -30,11 +34,11 @@ ELSE()
 ENDIF()
 
 if(NOT WIN32)
-    SET_UP_THIRD_PARTY(ADIOS2 ${LIB} include adios2 adios2_atl adios2_dill adios2_evpath adios2_ffs adios2_sst taustubs)
+    SET_UP_THIRD_PARTY(ADIOS2 ${LIB} include adios2_c adios2_atl adios2_dill adios2_evpath adios2_ffs adios2_taustubs adios2_cmenet adios2_cmzplenet adios2_cmepoll adios2_core adios2_cmfabric adios2_cmmulticast adios2_enet adios2_cmselect adios2_cxx11 adios2_cmsockets adios2_cmudp)
 else()
-    SET_UP_THIRD_PARTY(ADIOS2 ${LIB} include adios2 taustubs)
+    SET_UP_THIRD_PARTY(ADIOS2 ${LIB} include adios2_c adios2_taustubs)
 endif()
```

Comment:
It would be really interesting to find out if this works on Windows...

Reply:
Would you still like me to test this on Windows?

Reply:
Maybe a little bit further down the road though, perhaps when I take this PR out of draft.

Reply:
Just thought I would let you know: in between waiting for other builds and tests to run, I compiled c-blosc and adios2 2.7.1 on Windows.
For serial-only build of adios2 it produced these adios2 libraries: `adios2_c, adios2_core, adios2_cxx11, adios2_taustubs`
For a  parallel build: `adios2_c, adios2_c_mpi, adios2_core, adios2_core_mpi, adios2_cxx11, adios2_cxx11_mpi, adios2_taustubs` (so it appears to have built both the serial and parallel versions of the library during the parallel build which is good, it means I should have to do separate builds any more).

I tried compiling your branch with these libraries, but ran into a lot of link errors for the `EADIOS2Database_par` target.  Unresolved externals.  It could be I missed some cmake-configure swizzle, or it could be the some symbols from the library aren't being properly exported for Windows. I will do some digging at a later time.

Reply:
I figured out my issue. On Windows , for parallel-adios2, the ADIOS2 reader needs to link with non_mpi versions of the libraries as well as the _mpi versions. I hadn't modified the Parallel section in FindADIOS2.cmake.

Reply:
I'm still finishing the build so I can run your new test. But I think all is good. I can either send you the necessary changes to FindADIOS2.cmake for Windows, or I can change the file myself after your merge. I will know for certain after I test.

Reply:
adios2 test successful on Windows.

Reply:
Great! Thanks for checking that for me.